amazon-cloudwatch-container-insights/k8s-yaml-templates/cwagent-kubernetes-monitoring/otel-config.yml

```diff
@@ -0,0 +1,252 @@
+# create namespace
```

Comment:
should we add this to the Container Insights repo?

Reply:
Somehow, I would not agree on adding EKS ADOT Config for our repository based on two things:
* Even though ADOT Collector is the side agent, ADOT Collector has a supported EKS on their own. Therefore, it wasteful for us to support this too on EKS.
* In the future in moving to Unified Agent and also inherits ADOT framework, this would be not needed since converting to OTEL Metrics would be under the hood and ADOT Collector would not be used alongside with CWAgent anymore.
Therefore, I would consider removing it instead of adding it.

Reply:
I can remove this. I was only using this for testing against otel. We are removing otel from cw agent so this should not be here.